src/main/java/seedu/dukeofbooks/data/person/Person.java

```diff
@@ -9,8 +9,8 @@
 
 public class Person {
     public static final int DEFAULT_PHONE_NUMBER = 10000000;
-    private PersonName name;
-    private Phone phone;
+    private final PersonName name;
+    private final Phone phone;
```

Comment:
If we make it final, it means that the name and phone is bound to the object. This means no change to these attributes, maybe we should include it in the docs